### PR TITLE
improvement(reporting): Add argus links to email template and elastic

### PR DESCRIPTION
**Pull request by k0machi**
This commit adds links to argus to the elastic search index and email
templates, allowing users to find argus runs quickly from those places.

Fixes scylladb/qa-tasks#1490

### Testing

- [ ] [Jenkins](https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/alexey/job/alexey-argus-testing/100/)

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

**Comment by k0machi** [comment]
The links are unconditional, so they might not work all the time (if run wasn't reported to argus for example).

**Review by fruch — APPROVED** [review]
LGTM

File: sdcm/report_templates/results_base.html [hunk 1]
```diff
@@ -176,6 +176,7 @@ <h3>Test details</h3>
             {% endif %}
             {% if test_id %}
             <li>Test-id: {{ test_id }}</li>
+            <li><a href="https://argus.scylladb.com/tests/scylla-cluster-tests/{{ test_id }}">Link to argus</a></li>
```
Comment (vponomaryov) [inline sdcm/report_templates/results_base.html:179]: Typical argus link has `https://argus.scylladb.com/test/%test-id%` structure.
So, are you sure about the `https://argus.scylladb.com/tests/scylla-cluster-tests/{{ test_id }}` ?

Comment (k0machi) [reply sdcm/report_templates/results_base.html:179]: Forgot to mention that this change would need https://github.com/scylladb/argus/pull/478 to be merged as well to support this link format, but we can start writing them now, they will work once that PR is merged.

Comment (vponomaryov) [reply sdcm/report_templates/results_base.html:179]: Added comment there which may influence this PR: https://github.com/scylladb/argus/pull/478#discussion_r1793900485

**Review by soyacz — APPROVED** [review]
LGTM

**Pull request merged**